parser.go

```diff
@@ -35,6 +36,8 @@ type parserOptions struct {
 	UnescapeValueDoubleQuotes   bool
 	UnescapeValueCommentSymbols bool
 	PreserveSurroundedQuote     bool
+	Debug 						DebugFunc
```

Comment:
I think this field should also be called `DebugFunc`, because it is a function now.

Reply:
makes sense